docker/kopia-build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.17-alpine3.14 AS builder
+
+ARG kopiaBuildCommit
+ARG kopiaRepoOrg
+
+RUN apk add git
+
+WORKDIR /
+
+RUN git clone https://github.com/${kopiaRepoOrg}/kopia.git
+
+ENV GITHUB_REPOSITORY=https://github.com/${kopiaRepoOrg}/kopia
+
+WORKDIR /kopia
+
+# Build kopia binary from specific commit
+RUN git checkout ${kopiaBuildCommit} && \
+  wget https://github.com/goreleaser/goreleaser/releases/download/v1.11.2/goreleaser_1.11.2_x86_64.apk && \
+  apk add ./goreleaser_1.11.2_x86_64.apk --allow-untrusted && \
+  rm -f goreleaser_1.11.2_x86_64.apk && \
```

Comment:
@ankitjain235 we can copy kopia binary from the goreleaser docker image - https://hub.docker.com/r/goreleaser/goreleaser